tests/phpunit/unit-tests/functions/updates/class-llms-test-functions-updates-4140.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+* Test updates functions when updating to 4.14.0
+ *
+ * @package LifterLMS/Tests/Functions/Updates
+ *
+ * @group functions
+ * @group updates
+ * @group updates_4140
+ *
+ * @since [version]
+ * @version [version]
+ */
+class LLMS_Test_Functions_Updates_4140 extends LLMS_UnitTestCase {
+
+	private $sessions;
+
+	/**
+	 * Setup before class
+	 *
+	 * Include update functions file.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public static function setupBeforeClass() {
+		parent::setupBeforeClass();
+		require_once LLMS_PLUGIN_DIR . 'includes/functions/updates/llms-functions-updates-4140.php';
+	}
+
+	/**
+	 * Teardown the test case
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		// Clean posts and postmeta tables
+		global $wpdb;
+		$wpdb->query( "TRUNCATE TABLE {$wpdb->postmeta}" );
+		$wpdb->query( "TRUNCATE TABLE {$wpdb->posts}" );
+		// Delete transients.
+		delete_transient( 'llms_update_4140_remove_orphan_access_plans' );
+		delete_transient( 'llms_4140_skipper_orphan_access_plans' );
+	}
+
+	/**
+	 * Test llms_update_4140_remove_orphan_access_plans
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_update_4140_remove_orphan_access_plans() {
+
+		// Create orphan access plans.
+		$access_plan_ids = $this->factory->post->create_many(
+			10,
+			array(
+				'post_type' => 'llms_access_plan',
+			)
+		);
+		foreach ( $access_plan_ids as $access_plan_id ) {
+			update_post_meta( $access_plan_id, '_llms_product_id', 999 );
```

Comment:
At some point I believe we may end up in a scenario where this could cause failing tests. To ensure we have a truly non-existent post we could add 1 to the last created post id:

```suggestion
			update_post_meta( $access_plan_id, '_llms_product_id', end( $access_plan_ids ) + 1 );
```

Although I may be incorrect my hypothetical worry